clouddq/classes/dq_configs_cache.py

```diff
@@ -461,42 +515,42 @@ def _resolve_bigquery_entity_uri(
                 "metadata_default_registries YAML configs, e.g.\n"
                 f"{SAMPLE_DEFAULT_REGISTRIES_YAML}"
             )
-        required_arguments = ["projects", "lakes", "locations", "zones"]
+        required_arguments = ["projects", "datasets", "tables"]
         for argument in required_arguments:
             uri_argument = entity_uri.get_configs(argument)
             if not uri_argument:
                 raise RuntimeError(
-                    f"Failed to retrieve default Dataplex '{argument}' for "
+                    f"Failed to retrieve default Bigquery '{argument}' for "
                     f"entity_uri: {entity_uri.complete_uri_string}. \n"
                     f"'{argument}' is a required argument to look-up metadata for the entity_uri "
-                    "using Dataplex Metadata API.\n"
-                    "Ensure the BigQuery dataset containing this table "
-                    "is registered as an asset in Dataplex.\n"
-                    "You can then specify the corresponding Dataplex "
-                    "projects/locations/lakes/zones as part of the "
-                    "metadata_default_registries YAML configs, e.g.\n"
-                    f"{SAMPLE_DEFAULT_REGISTRIES_YAML}"
+                    "using Bigquery API.\n"
                 )
-        dataplex_entities_match = client.list_dataplex_entities(
-            gcp_project_id=entity_uri.get_configs("projects"),
-            location_id=entity_uri.get_configs("locations"),
-            lake_name=entity_uri.get_configs("lakes"),
-            zone_id=entity_uri.get_configs("zones"),
-            data_path=entity_uri.get_entity_id(),
-        )
-        logger.info(f"Retrieved Dataplex Entities:\n{pformat(dataplex_entities_match)}")
-        if len(dataplex_entities_match) != 1:
-            raise RuntimeError(
-                "Failed to retrieve Dataplex Metadata entry for "
-                f"entity_uri '{entity_uri.complete_uri_string}' "
-                f"with error:\n"
-                f"{pformat(dataplex_entities_match.json())}\n\n"
-                f"Parsed entity_uri configs:\n"
-                f"{pformat(entity_uri.to_dict())}\n\n"
-            )
-        dataplex_entity = dataplex_entities_match[0]
-        clouddq_entity = dq_entity.DqEntity.from_dataplex_entity(
-            entity_id=entity_uri.get_db_primary_key(),
-            dataplex_entity=dataplex_entity,
-        )
-        return clouddq_entity
+            else:
```

Comment:
this else clause is being run inside of the loop `for argument in required_arguments:`, which means that it is executed at least 3 times, and may fail if a uri has `projects` but not `dataset`.

Please take everything after the else clause out of this loop.